core/lib/ChromeCore.ts

```diff
@@ -93,9 +96,9 @@ export default class ChromeCore {
 
         const pages = await puppBrowser.pages();
         await Promise.all(pages.map(async x => x.close())).catch(error => {
-          log.warn('error closing initial puppeteer browser page', error);
+          log.warn('Error closing initial chrome browser page', error);
```

Comment:
Isn't this log.warn missing a sessionId arg?

And how was it not caught during typescript build? Or is `error` being treated as a string type?

Reply:
Yep, it's a bug. Error is type "any" in a catch, for some reason.